scripts/tests/test_suites.sh

```diff
@@ -143,7 +143,7 @@ for j in "${iter_array[@]}"; do
         echo "          * Pairing to device"
         "${test_case_wrapper[@]}" out/debug/standalone/chip-tool pairing qrcode "$node_id" MT:D8XA0CQM00KA0648G00 | tee "$pairing_log_file"
         echo "          * Starting test run: $i"
-        "${test_case_wrapper[@]}" out/debug/standalone/chip-tool tests "$i" "$node_id" "$delay" | tee "$chip_tool_log_file"
+        "${test_case_wrapper[@]}" out/debug/standalone/chip-tool tests "$i" "$node_id" delayInMs "$delay" | tee "$chip_tool_log_file"
```

Comment:
Would suggest renaming that arg to "--delayInMs" to make this look more like normal command-line args....